### ISSUE TITLE
Archive releases on Zenodo

### ISSUE THREAD

**Issue by Rekyt** [idiv-biodiversity/LCP#5]
Hi there,
To ease the citation process of the LCVP datasets (and also the lcvplants package), I would encourage you to archive the releases on Zenodo following the mechanisms offered by GitHub:
https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content
(you can follow this tutorial to do it: https://genr.eu/wp/cite/)

This would permanently archive the dataset on Zenodo and would automatically assign a DOI to each release of LCVP. It would ensure that whatever happens to GitHub in the future, the dataset could still be cited independently.

**Comment by BrunoVilela** [comment]
That is a great idea Matthias. However,  I currently do not have access to the idiv-biodiversity group as a collaborator, not sure who to ask for this access. Otherwise I cannot implement this. Do you know who I should contact?

**Comment by Rekyt** [comment]
Yes! I'm pinging @valentinitnelav to know if he can promote you to a role where you're allowed to do this for LCVP?

But I wonder if it's going to work on Zenodo side.

**Comment by BrunoVilela** [comment]
I have packages hosted in other organizations and Zenodo allows me use its functionalities, so this would probably work.
